rpc/client/cmds/jsonrpc.go

```diff
@@ -147,10 +129,28 @@ var (
 		Code:    -32603,
 		Message: "Internal error",
 	}
+	// ErrRPCDatabase indicates a database error.
+	ErrRPCDatabase = &RPCError{
+		Code:    -32604,
```

Comment:
should follow http://xmlrpc-epi.sourceforge.net/specs/rfc.fault_codes.php and https://www.jsonrpc.org/specification 5.1 Error object